src/main/scala/memeid.scala

```diff
@@ -5,18 +5,30 @@ import cats.kernel._
 import java.lang.Long
 import java.util.{UUID => JUUID}
 
-case class UUID(msb: Long, lsb: Long) 
+case class UUID(juuid: JUUID) {
+  @inline 
+  def msb: Long = juuid.getMostSignificantBits
+  @inline
+  def lsb: Long = juuid.getLeastSignificantBits
+}
 
 object UUID {
-  val empty: UUID = UUID(0, 0)
+  val empty: UUID = UUID(new JUUID(0, 0))
+
+  def apply(msb: Long, lsb: Long): UUID =
+    UUID(new JUUID(msb, lsb))
+
+  def toJava(u: UUID): JUUID =
```

Comment:
Wouldn't it be better if this method were in the case class? So we can use it like:

```scala
uuid.toJava
```